### PR TITLE
Spellcrafting ui

### PR DESCRIPTION
**Pull request by Sanko-Kallig**

### CODE REVIEW

**Review by Sven-dev — APPROVED** [review]

File: RPG Wizardry/Assets/Scripts/Player/Inventory/PlayerInventory.cs [hunk 1]
```diff
@@ -115,9 +121,16 @@ public bool UnlockSpell(SpellPage page)
             }
 
             return page.Unlocked;
-        } 
+        }
         #endregion
 
+        private void Start()
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/Player/Inventory/PlayerInventory.cs:127]: Summary

File: RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs [hunk 2]
```diff
@@ -36,33 +36,32 @@ public class DataManager : MonoBehaviour
         [SerializeField]
         private TextMeshProUGUI message;
 
+        [SerializeField]
+        private Texture2D splatTex;
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs:40]: summary + tooltip

File: RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs [hunk 3]
```diff
@@ -36,33 +36,32 @@ public class DataManager : MonoBehaviour
...
+
         /// <summary>
         /// Button the player can use to check if he did the research correctly.
         /// </summary>
         [SerializeField]
         private Button checkButton;
+
+        private bool started = false;
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs:48]: summary

File: RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs [hunk 4]
```diff
@@ -36,33 +36,32 @@ public class DataManager : MonoBehaviour
...
         #endregion
         #region Methods
-        // Start is called before the first frame update
-        void Start()
+
+        private void Start()
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs:52]: summary

File: RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs [hunk 5]
```diff
@@ -71,6 +70,17 @@ void Update()
            
         }
 
+        private void UpdateBars()
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/ResearchData/DataManager.cs:73]: s

File: RPG Wizardry/Assets/Scripts/Sorcery/SpellPageManager.cs [hunk 6]
```diff
@@ -14,36 +15,35 @@ public class SpellPageManager : MonoBehaviour
...
-        private PlayerInventory inventory;
-        /// <summary>
         /// Title of spell page
         /// </summary>
         [SerializeField]
         private TextMeshProUGUI title;
         [SerializeField]
         private Button button;
+        [SerializeField]
+        private Image spellImage;
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/Sorcery/SpellPageManager.cs:25]: s

File: RPG Wizardry/Assets/Scripts/Sorcery/SpellPageManager.cs [hunk 7]
```diff
@@ -14,36 +15,35 @@ public class SpellPageManager : MonoBehaviour
...
 
         #endregion
         #region Methods
-        private void Start()
+
+        public void SetSelectedSpell(SpellPage target)
```
Comment (frankvHoof93) [inline RPG Wizardry/Assets/Scripts/Sorcery/SpellPageManager.cs:30]: s

**Review by frankvHoof93 — APPROVED** [review]
Clean this up later

**Pull request merged**